spec/models/user_spec.rb

```diff
@@ -0,0 +1,21 @@
+# == Schema Information
+#
+# Table name: users
+#
+#  id                     :bigint           not null, primary key
+#  email                  :string           default(""), not null
+#  encrypted_password     :string           default(""), not null
+#  reset_password_token   :string
+#  reset_password_sent_at :datetime
+#  gender                 :string
+#  provider               :string           default("email"), not null
+#  uid                    :string           default(""), not null
+#  tokens                 :string
+#  created_at             :datetime         not null
+#  updated_at             :datetime         not null
+#
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
What about adding some test here? 👀 
Maybe to check if before validation is actually setting the `uid`.

Reply:
Thank you for the suggest, in the next pull request you are going to find the latest changes requested